topics:
  input:
    GNSS: "/anafi/gnss"
  output:
    waypoint: "/search/waypoint"

service:
  node_activation: "search_node_active"

services:
  distressed_coord: "distressed_coord"

camera:
  hfov: 69
  image_width: 1280
  image_height: 720

search:
  distressed_coord:
    long: 10.486565
    lat: 63.504124
  areasize:
    north: 250
    east: 250
  altitude: 10
  overlap: 0.15  

utilities:
  save_grid:
    subfolder_name: "grid"